### ISSUE TITLE
Error with the shebang on ubuntu 24.04

### ISSUE THREAD

**Issue by aseques** [MartinRybergLaude/bredbandskollen-cli#1]
I get an error related to the shebang used (due to the --no-warnings), it seems that's platform dependant (see **[here](https://stackoverflow.com/a/75536458/322049))**

```
$ bredbandskollen
/usr/bin/env: 'node --no-warnings': No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines

```

When removing the --no-warnings, it works as expected.

**Comment by MartinRybergLaude** [comment]
Interesting, thanks for reporting. I'll see if I can implement any of the alternatives mentioned in that thread.

**Comment by MartinRybergLaude** [comment]
Fixed in #2, published as version 1.0.3 latest on npm.
